README.md

```diff
@@ -227,7 +227,13 @@ This fastify plugin adds 3 utility decorators to your fastify instance using the
   - `refresh_token` (optional, only if the `offline scope` was originally requested, as seen in the callbackUriParams example)
   - `token_type` (generally `'bearer'`)
   - `expires_in` (number of seconds for the token to expire, e.g. `240000`)
-- `getNewAccessTokenUsingRefreshToken(refreshToken, params, callback)`: A function that takes a refresh token and retrieves a new `AccessToken` object. This is generally useful with background processing workers to re-issue a new token when the original token has expired. The `params` argument is optional and it is an object that can be used to pass in extra parameters to the refresh request (e.g. a stricter set of scopes). If the callback is not passed this function will return a promise. The object resulting from the callback call or the promise resolution is a new `AccessToken` object (see above).
+- `getNewAccessTokenUsingRefreshToken(AccessToken, params, callback)`: A function that takes a AccessToken and retrieves a new `AccessToken` object. This is generally useful with background processing workers to re-issue a new token when the original token has expired. The `params` argument is optional and it is an object that can be used to pass in extra parameters to the refresh request (e.g. a stricter set of scopes). If the callback is not passed this function will return a promise. The object resulting from the callback call or the promise resolution is a new `AccessToken` object (see above). Example of how you would use it for `name:googleOAuth2`:
```

Comment:
This does not look right.

Should be more like:

```suggestion
- `getNewAccessTokenUsingRefreshToken(Token, params, callback)`: A function that takes a `Token`-Object and retrieves a new `Token`-Object. This is generally useful with background processing workers to re-issue a new AccessToken when the previous AccessToken has expired. The `params` argument is optional and it is an object that can be used to pass in additional parameters to the refresh request (e.g. a stricter set of scopes). If the callback is not passed this function will return a Promise. The object resulting from the callback call or the resolved Promise is a new `Token` object (see above). Example of how you would use it for `name:googleOAuth2`:
```

Reply:
The `Token` - object is actually an AccessToken object as mentioned here: [AccessToken](https://github.com/lelylan/simple-oauth2/blob/master/API.md#accesstoken).  `getNewAccessTokenUsingRefreshToken(Token, params, callback)` is okay. Other changes can be made as `AccessToken`-Object

```suggestion
- `getNewAccessTokenUsingRefreshToken(Token, params, callback)`: A function that takes a `AccessToken`-Object as `Token` and retrieves a new `AccessToken`-Object. This is generally useful with background processing workers to re-issue a new AccessToken when the previous AccessToken has expired. The `params` argument is optional and it is an object that can be used to pass in additional parameters to the refresh request (e.g. a stricter set of scopes). If the callback is not passed this function will return a Promise. The object resulting from the callback call or the resolved Promise is a new `AccessToken` object (see above). Example of how you would use it for `name:googleOAuth2`:
```

Reply:
@Uzlopak Does the new suggested change looks right ?